src/components/SimpleBlock.vue

```diff
@@ -0,0 +1,271 @@
+              </svg>
+            </span>
+            <span>Alert</span>
+          </label>
+        </div>
+      </div>
+
+      <div class="tabs__content-wrapper">
+        <div class="tabs__content standard" v-if="activeTab === 'standard'">
+          <!-- Контент вкладки стандарт -->
+          <form
+            action="#"
+            @submit="checkForm"
+          >
+            <div class="standard__input-wrapper">
+              <label for="button-link">Link to the button</label>
+              <input
+                v-model="link"
+                name="button-link"
+                id="button-link"
+                type="text"
+              >
+            </div>
+            <p class="standard__text"> You can use any link, but it’s length should not exceed 200 characters</p>
+
+            <p v-if="errors.length">
+              <b>Пожалуйста исправьте указанные ошибки:</b>
+              <ul>
+                <li v-for="error in errors" :key="error">{{ error }}</li>
+              </ul>
+            </p>
+
+            <button class="standard__submit" type="submit">Continue</button>
+          </form>
+        </div>
+
+        <div class="tabs__content" v-if="activeTab === 'event'">
+          Контент вкладки event
+        </div>
+
+        <div class="tabs__content" v-if="activeTab === 'alert'">
+          <!-- Контент вкладки alert -->
+        </div>
+      </div>
+    </div>
+
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'SimpleBlock',
+  data () {
+    return {
+      tabs: {
+        standard: {
+          disabled: false
+        },
+        event: {
+          disabled: false
+        },
+        alert: {
+          disabled: true
+        }
+      },
+      activeTab: 'standard',
+      errors: [],
+      link: null
+    }
+  },
+  methods: {
+    changeTab (tab) {
+      if (!this.tabs[tab].disabled) {
+        this.activeTab = tab
+      }
+    },
+    checkForm: function (e) {
+      e.preventDefault()
+      this.errors = []
+
+      if (this.link.length >= 200) {
+        this.errors.push('Длина ссылки превышает 200 символов.')
+      }
+
+      if (this.errors.length === 0) {
+        console.log(this.link)
+      }
+    }
+  }
+}
+</script>
+
+<style scoped>
+.card {
+  width: 512px;
+  min-height: 480px;
+  margin: 0 auto;
+  padding: 36px 25px 80px 32px;
+  border-radius: 4px;
+  box-shadow: 0 8px 8px 0 rgba(119, 131, 143, 0.16);
+  background-color: #fff;
+  box-sizing: border-box;
+}
+.card__title {
+  margin: 0 2px 0 0;
+  font-size: 24px;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
```

Comment:
this is default styles. this is no point to declare it. but you could declare only in parent DOM element